db/migrate/20230303153213_allow_nil_invoice_for_fees.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AllowNilInvoiceForFees < ActiveRecord::Migration[7.0]
+  def change
+    change_column_null :fees, :invoice_id, null: true
```

Comment:
Hum this is already the case no? 😅 
Nothing changed in the schema...

Reply:
Oh yeah indeed! :smile: 

Weird... Fixing it!